lib/rototiller/task/collections/command_collection.rb

```diff
@@ -0,0 +1,14 @@
+require 'rototiller/task/collections/param_collection'
+require 'rototiller/task/params/command'
+
+module Rototiller
+  module Task
+
+    class CommandCollection < ParamCollection
+      def allowed_class
+        Command
+      end
```

Comment:
We could just do an `attr_reader` on PramCollection and assign the instance variable in an `initialize`.  We are probably splitting hairs here thought.